web/app/js/components/Navigation.jsx

```diff
@@ -122,6 +124,9 @@ const styles = theme => {
       fontSize: "18px",
       paddingLeft: "3px",
       paddingRight: "3px",
+    },
+    badge: {
+      backgroundColor: "#ffeb3b",
```

Comment:
consider using material ui colors (i think it's equivalent)?
```diff
diff --git a/web/app/js/components/Navigation.jsx b/web/app/js/components/Navigation.jsx
index cc0bee28..5457bc26 100644
--- a/web/app/js/components/Navigation.jsx
+++ b/web/app/js/components/Navigation.jsx
@@ -30,6 +30,7 @@ import Version from './Version.jsx';
 import classNames from 'classnames';
 import { withContext } from './util/AppContext.jsx';
 import { withStyles } from '@material-ui/core/styles';
+import yellow from '@material-ui/core/colors/yellow';

 const styles = theme => {
   const drawerWidth = theme.spacing.unit * 31;
@@ -126,7 +127,7 @@ const styles = theme => {
       paddingRight: "3px",
     },
     badge: {
-      backgroundColor: "#ffeb3b",
+      backgroundColor: yellow[500],
     }
   };
 };
```

Reply:
Thanks, am now using the color library from Material UI ✅